### PR TITLE
WIP -  implementation of routeos_interface_ethernet

### PR DESCRIPTION
**Pull request by jlpedrosa**
Hi! 

I was wondering if you'd be open for PRs? it is related to the issue I opened ( #255 ). I just wanted to drop a WIP to grock if you are interested in contributions and/or if it's more or less where your head is at. 

There are a lot of properties pending, and the tests. Just to see what is your opinion.

### CODE REVIEW

**Comment by vaerh** [comment]
Of course, I'm always happy to have good, clean code.
Please supplement PR with tests.

File: docs/resources/interface_ethernet.md
Comment (vaerh) [inline docs/resources/interface_ethernet.md:1]: Documentation is automatically generated with each release. For this reason, there is no need to generate documentation in PR. But it is necessary to fill in the [resource example and export example files](https://github.com/terraform-routeros/terraform-provider-routeros/tree/main/examples/resources).

**Comment by vaerh** [comment]
Please check the current changes with yourself. The tests should run without warnings.

**Comment by jlpedrosa** [comment]
@vaerh let me know if you want any changes, happy to do them. Also, I ran the test locally, but actions is not enabled for this PR (authorization), so I'm not sure if it's going to work in all CI scenarios.

**Review by vaerh — APPROVED** [review]

**Pull request merged**